pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "manim"
+version = "0.2.0"
+description = "Animation engine for explanatory math videos."
+authors = ["3b1b <grant@3blue1brown.com>"]
```

Comment:
issue: need to change this

Reply:
What should I do?

Reply:
Ehrm. Not sure here, but 3b1b shouldn't be the only one listed. (He should be there, but not alone.)

@eulertour ideas?

Reply:
Why not say `The Manim Community Developers` like we do in `LICENSE.community? We don't absolutely need to include an EMail Address right?

Another option would be to include the usernames/emails of everyone on the Core dev team, if they're willing, but that would leave out the non-core devs.

Reply:
Maybe https://bugs.python.org/msg93284 idea is nice and can help.

Reply:
Yeah, we should setup a common email account at some point in the future.

How about `authors=['3b1b', 'The Manim Community Developers']`? Using only 3b1b's email but not any of ours would be confusing.

Reply:
We can create create a mailing list using Google groups of even now.

Reply:
If this is decided than yes we are gonna merge this.

Reply:
This question does not seem resolved, +1 for the suggestion that @leotrs made.

```suggestion
authors = ["3b1b", "The Manim Community Developers"]
```